README.md

```diff
@@ -78,7 +78,16 @@ $ make openapi
 
 ### Configuration
 
-The default configuration file in ./configs/config.yaml.example shows all available config options.  Any of these can be overridden with an environment variable.  For example  "database.name" can be passed in via an environment variable named "DATABASE_NAME".
+The default configuration file in `./configs/config.yaml.example` shows all available config options. Any of these can be overridden with an environment variable. For example `database.name` can be passed in via an environment variable named `DATABASE_NAME`.
+
+Actually some settings are included for the workspace into the
```

Comment:
```suggestion
Actually some settings are included for the workspace into the
```
This sentence makes no sense. Maybe you mean something like:
Some settings are included for the workspace in the same configuration file
used from the CLI (./configs/config.yaml); this lets

Reply:
Thanks! Restoring the previous state! Wrong commit for this pr. I started to update it for https://github.com/content-services/content-sources-backend/pull/40